cmd/virt-handler/virt-handler.go

```diff
@@ -312,35 +328,6 @@ func (app *virtHandlerApp) Run() {
 	defer close(stop)
 	factory.Start(stop)
 
-	selinuxLauncherType := clusterConfig.GetSELinuxLauncherType()
```

Comment:
do we not support custom selinux labels for virt-launcher now? maybe that's fine. I don't understand why this existed in the first place.

Reply:
As I understand this, the problem was that we didn't manage to install (use `semanage`) on RHCOS / FCOS because this binary wasn't present. As a workaround, we would allow the installation upstream, but on RHCOS we would use a custom policy and that would prevent the installation.
With this PR, we are no longer dependent on python binaries and will use the native selinux library, which will make this workaround irrelevant.

Going forward, we should remove our custom selinux policy completely, I think @jean-edouard is working on a PR.

Reply:
The setting of the customer policy is still happening in https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-controller/services/template.go#L988

Reply:
But we still support our default policy. So istn't removing the `clusterConfig.GetSELinuxLauncherType()` happening due to the fact that the  PR is older than allowing overriding the type? I would expect that we keep this in this PR.